### DISCUSSION TITLE
Autogeneration of Bibliography

### DISCUSSION THREAD

**Discussion by mlnomadpy** [marp-team/marp#91] [Q&A] [answered]
Is there a way to autogenerate the bibliography of cited books and web pages at the end of the presentation.
I have tried to use the syntax from [R Markdown ](https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html) but it seems not to be working.
Is it possible with marp?
Thank you very much.

**Comment by yhatt** [comment, marked as answer]
Marp is only following [CommonMark](https://commonmark.org/) and a few GFM features. Basically _[Marp team won't work to support out of CommonMark specification.](https://github.com/marp-team/.github/blob/master/CONTRIBUTING.md#before-adding-new-feature)_ You cannot use many R Markdown features because they are not part of CommonMark.

We always recommend to extend by Marpit/markdown-it plugins as you like. Marp CLI can use the plugged engine: https://github.com/marp-team/marp-cli#functional-engine

But unfortunately, there is no plugins for bibliography currently:

- markdown-it/markdown-it#55
- marp-team/marpit#282
